docgen/src/gettingstarted.md

```diff
@@ -6,49 +6,323 @@ nav_groups:
+## Welcome to react-instantsearch
 
-**This is an alpha release, everything may be broken**.
+React-instantsearch is the ultimate toolbox for creating instant search
+experience using React.js and Algolia.
 
-Links:
-- [Chat with us](https://gitter.im/algolia/instantsearch.js) on gitter, we are active and waiting for feedback
-- Code is at <https://github.com/algolia/instantsearch.js/tree/v2>, jump in
+In this tutorial, you'll learn how to:
 
+ - add react-instantsearch in your react project
+ - bootstrap react-instantsearch
+ - display results from Algolia
+ - add components to filter the results
+ - connect your own component to the search
 
-## Example
+## Before we start
+
+As mentionned earlier react-instantsearch is meant to be used with Algolia.
+
+Therefore, you'll need the credentials to an Algolia index. There are two
+ways to follow this `getting started`:
+ - using your own account with our dataset
+ - using our demo index credentials
+
+If you want to use your account, you should have the folowing information:
+ - `appId`, the application id
+ - `searchKey`, the search key to your application
+ - `indexName`, the name of the index in which you uploaded the data
+
+In case you just want to try the library here are the credentials for
+our sample dataset:
+ - `appId`: `latency` 
+ - `searchKey`: `3d9875e51fbd20c7754e65422f7ce5e1`
+ - `indexName`: `bestbuy`
+
+This guide expects you to have a working react.js project. If
+you want to start from scratch, we suggest you use
+[create-react-app](https://github.com/facebookincubator/create-react-app#getting-started).
+
+## Install react-instantsearch
+
+react-instantsearch is available on NPM. To install it you just need
+to type this command in your shell:
 
 ```sh
-npm install react-instantsearch --save
+npm install --save react-instantsearch
 ```
 
-app.js
-```js
+## Add the InstantSearch wrapper
+
+The instantsearch wrapper is the component that will connect to Algolia
+and that will synchronise all the widgets together. It maintains the state
+of the search, do the queries and provide the results to the widgets so
+that they can update themselves if needed.
+
+```javascript
 import React from 'react';
-import ReactDOM from 'react-dom';
 
-import {
-  InstantSearch,
-  SearchBox,
-  Hits,
-} from 'react-instantsearch';
+import {InstantSearch} from 'react-instantsearch';
 
-const App = () =>
+var App = () =>
   <InstantSearch
-    appId="latency"
-    apiKey="6be0576ff61c053d5f9a3225e2a90f76"
-    indexName="movies"
+    appId={appId}
+    apiKey={apiKey}
+    indexName={indexName}
   >
-    <div>
-      <SearchBox />
+   // The instantsearch components will go here
+  </InstantSearch>
+
+// Needed only if you're js app doesn't do it already.
+// Create-react-app does it for you ;)
+React.render(<App />, document.querySelector('#app'));
+```
+
+As you can see, there are a few mandatory props. Those props are the
```

Comment:
`appId, apiKey and indexName are mandatory props. Those props..` instead of `As you can see` (yes I can :D)